src/less/star-rating-select/star-rating-select.stories.js

```diff
@@ -1,47 +1,47 @@
 export default { title: 'Star Rating Select/Star Rating Select' };
 
 export const base = () => `
-<div class="star-rating-select">
-    <span class="radio">
-        <input aria-label="1 Star" class="radio__control" type="radio" name="radio-star-rating-select" value="1">
-        <span class="radio__icon">
-            <svg class="star-rating-select__icon" focusable="false">
+<div role="radiogroup" aria-label="Rate this product" class="star-rating-select">
+    <span class="star-rating-select__radio">
+        <input aria-label="1 Star" type="radio" name="radio-star-rating-select" value="1">
+        <span class="star-rating-select__radio__icon">
+            <svg focusable="false">
                 <use xlink:href="#icon-star-dynamic"></use>
             </svg>
         </span>
     </span>
 
-    <span class="radio">
-        <input aria-label="2 Stars" class="radio__control" type="radio" name="radio-star-rating-select" value="2">
-        <span class="radio__icon">
-            <svg class="star-rating-select__icon" focusable="false">
+    <span class="star-rating-select__radio">
+        <input aria-label="2 Stars" type="radio" name="radio-star-rating-select" value="2">
+        <span class="star-rating-select__radio__icon">
```

Comment:
I believe this one can just be `star-rating-select__radio-icon`

Reply:
I think you're looking at an outdated file. Line 16 currently reads,

`<span class="star-rating-select__icon">`